### PR TITLE
added stopword functionality in zingg FieldDefinition

### PR DESCRIPTION
**Pull request by RavirajBaraiya**
added setStopWordsCutoff method in zingg.Arguments class
added setStopWords method in zingg.FeildDefinition class
added stopword parameter in one field of amazon-google and iTunes-amazon examples

### CODE REVIEW

**Comment by RavirajBaraiya** [comment]
issue #340

**Comment by RavirajBaraiya** [comment]
issue #252 
added new config files for "Amazon-google" dataset involving stop word removal.

File: examples/stopWords.csv [hunk 1]
```diff
@@ -0,0 +1,16 @@
+StopWord
```
Comment (sonalgoyal) [inline examples/stopWords.csv:1]: move this file into amazon google and update code and config accordingly

**Review by sonalgoyal — CHANGES_REQUESTED** [review]
move the stop words file under the relevant example folder and use from there.

File: examples/stopWords.csv (hunk 1 above)
Comment (RavirajBaraiya) [reply examples/stopWords.csv]: moved csv to amazon-google

**Pull request merged**